### PR TITLE
Change slaves to replicas

### PR DESCRIPTION
**Pull request by maxiloc**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /vendor
 composer.lock
 phpunit.xml
+.idea/
```
Comment (JanPetr) [inline .gitignore]: Isn't it better to this one into global gitignore as it's related to specific dev env?

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ This PHP package integrates the Algolia Search API to the Laravel Eloquent ORM.
 4. [Ranking & Relevance](#ranking--relevance)
 5. [Options](#options)
 6. [Indexing](#indexing)
-7. [Master/Slave](#masterslave)
+7. [Primary/Replica](#masterreplica)
```
Comment (JanPetr) [inline README.md]: Why not `#primaryreplica`?

File: src/ModelHelper.php [hunk 3]
```diff
@@ -80,9 +80,16 @@ public function getSettings(Model $model)
         return property_exists($model, 'algoliaSettings') ? $model->algoliaSettings : [];
     }
 
-    public function getSlavesSettings(Model $model)
+    public function getReplicasSettings(Model $model)
```
Comment (JanPetr) [inline src/ModelHelper.php]: As this was `public` method, you should still keep it present:
```php
public function getSlavesSettings(Model $model)
{
    // trigger deprecated notice/warning

    return $this->getReplicasSettings(Model $model);
}
```

File: .gitignore (hunk 1 above)
Comment (rayrutjes) [inline .gitignore]: I'll go for the global ignore as well ;)

**Pull request merged**